File: 02/AED2/TrabalhosTeoricos/02_ProcessoAEDsII/03_Cometa/Cometa.java
import java.util.Scanner;

public class Cometa {
	public static void main(String args[]) {
		Scanner s = new Scanner(System.in);
		int year, next, term = 75;

		while (true) {
			year = s.nextInt();
			if (year == 0)
				break;
			for (next = 1986; next <= year; next += term)
				;
			System.out.println(next + 2);
		}
	}
}

File: 02/AED2/TrabalhoTeorico02_ProcessoAEDsII/03_Cometa/Cometa.c
#include <stdio.h>

int main(char **args){
	int year, next, term = 75;
	while(1){
		scanf("%d", &year);
		if(year == 0)
			return 1;
		for(next = 1986; next <= year;next += term);
		printf("%d\n", next+2);
	}	
}
